internal/subshell/bash/bash.go

```diff
@@ -219,3 +220,17 @@ func (v *SubShell) IsAvailable() bool {
 	}
 	return fileutils.FileExists(rcFile)
 }
+
+func (v *SubShell) TurnOffEcho() {
+	if runtime.GOOS == "windows" {
+		return // not supported
```

Comment:
Windows does not have access to the termios functions needed to disable echo in emulated shells like Git Bash (which is not interactive anyway).